.gitpod.yml

```diff
@@ -13,16 +13,13 @@ ports:
 
 tasks:
   - name: Forem Server
-    before: |
-      redis-server &
-    init: >
-      gitpod-init.sh
+    before: redis-server &
+    init: ./gitpod-init.sh
```

Comment:
Forgot the `./`. 🤦🏻‍♂️

Reply:
now I wonder how long that had been broken?

Reply:
A week. That is all on me since I'm the one that introduced that script. 😬